### PR TITLE
Upgrade Cosmos Hub to v11

### PR DESCRIPTION
**Pull request by reversesigh**
Prop 804
Height 16596000
Estimated Time:  Aug 16th 2023, 09:34:15 EST

https://www.mintscan.io/cosmos/proposals/804

### CODE REVIEW

**Comment by reversesigh** [comment]
@JeremyParish69 can we add `"windows/arm64"`` or should we just remove it for now?

**Comment by JeremyParish69** [comment]
> @JeremyParish69 can we add `"windows/arm64"`` or should we just remove it for now?

Ya let's add it

**Comment by dylanschultzie** [comment]
@JeremyParish69 I'm not sure what I'm missing here, as `"windows/arm64"` is now added: https://github.com/cosmos/chain-registry/pull/2608

**Comment by JeremyParish69** [comment]
> @JeremyParish69 I'm not sure what I'm missing here, as `"windows/arm64"` is now added: #2608

Just had to re-run post-fix. Should be good to go

File: cosmoshub/chain.json [hunk 1]
```diff
@@ -93,6 +94,31 @@
...
+        },
+        "binaries": {
+          "linux/amd64": "https://github.com/cosmos/gaia/releases/download/v11.0.0/gaiad-v11.0.0-linux-amd64",
+          "linux/arm64": "https://github.com/cosmos/gaia/releases/download/v11.0.0/gaiad-v11.0.0-linux-arm64",
+          "darwin/amd64": "https://github.com/cosmos/gaia/releases/download/v11.0.0/gaiad-v11.0.0-darwin-amd64",
+          "darwin/arm64": "https://github.com/cosmos/gaia/releases/download/v11.0.0/gaiad-v11.0.0-darwin-arm64",
+          "windows/amd64": "https://github.com/cosmos/gaia/releases/download/v11.0.0/gaiad-v11.0.0-windows-amd64.exe",
+          "windows/arm64": "https://github.com/cosmos/gaia/releases/download/v11.0.0/gaiad-v11.0.0-windows-arm64.exe"
+        },
         "next_version_name": ""
```
Comment (mmulji-ic) [inline cosmoshub/chain.json:122]: This will be v12

File: cosmoshub/chain.json [hunk 2]
```diff
@@ -93,6 +94,31 @@
...
+          "type": "cometbft",
+          "version": "v0.34.29"
+        },
+        "binaries": {
+          "linux/amd64": "https://github.com/cosmos/gaia/releases/download/v11.0.0/gaiad-v11.0.0-linux-amd64",
+          "linux/arm64": "https://github.com/cosmos/gaia/releases/download/v11.0.0/gaiad-v11.0.0-linux-arm64",
+          "darwin/amd64": "https://github.com/cosmos/gaia/releases/download/v11.0.0/gaiad-v11.0.0-darwin-amd64",
+          "darwin/arm64": "https://github.com/cosmos/gaia/releases/download/v11.0.0/gaiad-v11.0.0-darwin-arm64",
+          "windows/amd64": "https://github.com/cosmos/gaia/releases/download/v11.0.0/gaiad-v11.0.0-windows-amd64.exe",
+          "windows/arm64": "https://github.com/cosmos/gaia/releases/download/v11.0.0/gaiad-v11.0.0-windows-arm64.exe"
```
Comment (mmulji-ic) [inline cosmoshub/chain.json:120]: All the SHA's are missing, see [SHA256SUMS-v11.0.0.txt](https://github.com/cosmos/gaia/releases/download/v11.0.0/SHA256SUMS-v11.0.0.txt)

**Pull request merged**